05_customizing-extending-masa/06_events/02_lifecycles.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Lifecycles
+nav_order: 2
+grand_parent: Customizing/Extending Masa
+parent: Events
+last_modified_date: 11-01-2021 10:00
+permalink: /customizing-extending-masa/events/lifecycles/
+---
+
+# Lifecycle
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+- TOC
+{:toc}
+
+## Front-end request lifecycle
+
+```
+onGlobalRequestStart
```

Comment:
When rendered by MarkDown this just looks like one blob of code, altough be it indented.
Could it be an idea to make this a list with list items and children instead, to create more order and overview. Names of event can still be shown as code right and comments can be normal text.